<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiDBoxFEBase</class>
 <widget class="QMainWindow" name="UiDBoxFEBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DBoxFE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#UiDBoxFEBase {
  background-color: #ededed;
}

QAbstractScrollArea {
 background: transparent;
}</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="mCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mStackedWidgetOption">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="asaal::Welcome" name="mPageWelcome"/>
      <widget class="asaal::Profiles" name="mPageProfiles"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>asaal::Welcome</class>
   <extends>QWidget</extends>
   <header>welcome.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>asaal::Profiles</class>
   <extends>QWidget</extends>
   <header>profiles.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
